burger/toppings/identify.py

```diff
@@ -308,7 +315,7 @@ class IdentifyTopping(Topping):
         "identify.metadata",
         "identify.nbtcompound",
         "identify.nethandler.client",
-        "identify.nethandler.server",
+        "identify.nethandler.handshake",
```

Comment:
Looks good apart from this; you should probably add an additional `"identify.nethandler.server"` for the new `nethandler.server` one.

Though, it's not super important as we don't actually do anything on a finer-grained level than just using all of these to identify `IdentifyTopping`. Perhaps in the future it can be used so that if some parts fail/aren't found, then only things that depend on those parts are skipped, but that's not how it works currently.